.github/workflows/solidity-foundry.yml

```diff
@@ -285,3 +562,16 @@ jobs:
         working-directory: contracts
         env:
           FOUNDRY_PROFILE: ${{ matrix.product.name }}
+
+  check-fmt-results:
```

Comment:
new job that checks for results of the formatting matrix, we will set it as a required job